app/assets/stylesheets/avalon/_nav.scss

```diff
@@ -92,6 +94,8 @@ $state-info-border:              darken(adjust-hue($state-info-bg, -10), 7%) !de
 
 .navbar-header {
   @include media-breakpoint-down(md) {
+    width: 100%;
```

Comment:
I think this is not necessary at this screen size, since the `.navbar-header` is a hidden div at this point. Setting width to 100% pushes the visible nav items to the right. So, it works as expected at this screen width without setting this.

Instead I think `width: 100%` could be applied to the next breaking point -> `sm`

Reply:
Moving that line down to the next block and adjust it to `sm` from `xs` seems to have fixed a number of problem screen sizes.  Thanks for the suggestion!